### PR TITLE
Fix TODOs

### PR DESCRIPTION
**Pull request by stephanie-eng**
Depends on #4 for formatting changes.

Fix commented TODOs.

### CODE REVIEW

File: include/ros2_control_abb_driver/abb_system_position_only.hpp [hunk 1]
```diff
@@ -68,6 +68,7 @@ class ABBSystemPositionOnlyHardware : public hardware_interface::SystemInterface
 private:
   // Hardware parameters
   double robotstudio_port_;
```
Comment (AndyZe) [inline include/ros2_control_abb_driver/abb_system_position_only.hpp]: Should this really be a double? Seems like it should be a `size_t`

File: src/abb_system_position_only.cpp [hunk 2]
```diff
@@ -29,6 +29,9 @@ using namespace std::chrono_literals;
 
 namespace ros2_control_abb_driver
 {
+
+static constexpr auto DEG_TO_RAD = 180 / M_PI;
```
Comment (AndyZe) [inline src/abb_system_position_only.cpp:33]: Please put this in an anonymous namespace so it doesn't pollute the `ros2_control_abb_driver` namespace. An anonymous namespace keeps the constant local to this file. Also, I've found out from Tyler recently, the `static` does not do anything here.

```
namespace
{
  constexpr auto DEG_TO_RAD = 180 / M_PI;
}
```

File: src/abb_system_position_only.cpp [hunk 3]
```diff
@@ -233,7 +223,7 @@ return_type ABBSystemPositionOnlyHardware::read()
 {
   RCLCPP_INFO(rclcpp::get_logger("ABBSystemPositionOnlyHardware"), "read()");
 
-  abb::egm::wrapper::Input tmp_input;  // TODO(seng): function no longer const, change this
+  abb::egm::wrapper::Input tmp_input;
```
Comment (AndyZe) [inline src/abb_system_position_only.cpp]: Rename to `temp_input`, just to stay consistent

Same for `temp_current_positions` below

File: src/abb_system_position_only.cpp (hunk 2 above)
Comment (stephanie-eng) [reply src/abb_system_position_only.cpp:33]: I'm pretty sure static / anonymous namespace do the same thing (see [here](https://stackoverflow.com/questions/154469/unnamed-anonymous-namespaces-vs-static-functions)). My understanding is that static variables are translation unit local, so shouldn't pollute the namespace (hence why `static` in an anonymous namespace is redundant, and a no-op). On the other hand, if we in general prefer anonymous namespaces for stylistic reasons, I'm fine with changing it.

Comment (AndyZe) [reply src/abb_system_position_only.cpp:33]: OK, good point. Leave it as-is, then

**Review by AndyZe — APPROVED** [review]

**Pull request merged**